src/controllers/crop_controller.py

```diff
@@ -74,7 +75,7 @@ def setup_crop_positions(self):
         curr_row = 0
         curr_crop_type = 0
         curr_crop = 0
-        location = [0, 0, 0]
+        location = [-20, 0, 0]
```

Comment:
Does -20 work for every row and column combination?

Reply:
So this currently, works for over 25 crops. Ideally, we start manipulating the camera to auto center to the crops

Reply:
Could we change x location as a variable that changes based on number of crops and crop rows? Eg, 50 crops with 5 rows would have x = -5*row width as center instead of -20?

Reply:
Good idea! Will do that